src/user-rank/user-rank.module.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Module } from '@nestjs/common';
+import { GraphileWorkerModule } from '../graphile-worker/graphile-worker.module';
+import { PrismaModule } from '../prisma/prisma.module';
+import { UserRankJobController } from './user-rank.job.controller';
+import { UserRankJobService } from './user-rank.job.service';
+import { UserRankService } from './user-rank.service';
+
+@Module({
+  exports: [UserRankService, UserRankJobService],
+  imports: [PrismaModule, GraphileWorkerModule],
+  providers: [UserRankService, UserRankJobService],
+  controllers: [UserRankJobController],
+})
+export class UserRankModule {}
```

Comment:
Can we rename the file?
```suggestion
export class UserRanksModule {}
```